index.js

```diff
@@ -513,15 +546,17 @@ class KeyringController extends EventEmitter {
    * encrypts that array with the provided `password`,
    * and persists that encrypted string to storage.
    *
-   * @param {string} password - The keyring controller password.
    * @returns {Promise<boolean>} Resolves to true once keyrings are persisted.
    */
-  async persistAllKeyrings(password = this.password) {
-    if (typeof password !== 'string') {
-      throw new Error('KeyringController - password is not a string');
+  async persistAllKeyrings() {
+    const { encryptionKey } = this.memStore.getState();
+
+    if (!this.password && !encryptionKey) {
+      throw new Error(
+        'Cannot persist vault without password and encryption key',
```

Comment:
Realizing I'm probably late to the party here but: How ingrained is the use of the word "password" in user-facing messages wrt this passphrase?

The reason why I'd like to be meticulous and (however late) make clear distinctions is to help users set expectations:
* A `password` is validated against some source of truth to perform authorization. There is always *some* way to reset it to regain access.
* A `passphrase` is used to derive an encryption key. If it's lost it and there are no backups, there is no recourse.

Reply:
I'm happy to update to either word, but I don't think it should block this PR.